content/devdocs/references/p2p_networking.md

```diff
@@ -71,7 +71,12 @@ The following example is an annotated hex dump of a mainnet message
 header from an [`GETP` message][getp message] which has no payload.
 
 {{ <highlight text> }}
-TODO : GETP message hex dump
+GetPeersMessage:
+```
+04 00 00 00 00 00 00 00 ........................... Full message
```

Comment:
No need to add full message . It's too much , especially if messages are long. Annotated message would be enough. Please remove